### ISSUE TITLE
Import HTML regex breaks if newlines in title

### ISSUE THREAD

**Issue by sottwell** [modxcms/revolution#12937]
### Summary
When importing an HTML document the regex that parses the title tags breaks if there are newlines in the title.

### Step to reproduce
Add some extra lines to the title tag in an HTML page's head. Import the HTML document.

### Observed behavior
The title will not be used as the resource's pagetitle, it will default to the base filename.

### Expected behavior
It should use the HTML title. In order to do so the preg_match needs to use the s modifier on line 104 of core/model/modx/import/modstaticimport.class.php:
```
if (preg_match("/<title>(.*)<\/title>/siU", $file, $matches)) {
```
Now it will use the HTML title as expected, even with extra lines in the title.

### Environment
N/A

**Comment by Jako** [comment]
It would be better to change the regex to trim the whitespace away.

```
if (preg_match("/<title>\s*(.*)\s*<\/title>/siU", $file, $matches)) {
```

**Comment by sottwell** [comment]
Whatever works. Regular expressions are not my strong point.

**Comment by Mark-H** [comment]
Fixed in #12941
